sources/TileDB.CSharp/VFS.cs

```diff
@@ -391,6 +427,38 @@ public void VisitChildren<T>(string uri, Func<string, T, bool> callback, T callb
         callbackData.Exception?.Throw();
     }
 
+    /// <summary>
+    /// Visits all files of a directory and its subdirectories recursively.
```

Comment:
Might be worth mentioning recursive listing is only supported on S3 since this could be the first docs a user reads.

Reply:
Added a note.